### PR TITLE
Fix AppWrapper + Avatar Typography + NavBar positioning

### PR DESCRIPTION
**Pull request by lucianojss**
# [#201](https://digitalrig.atlassian.net/browse/BUD-201) Fix AppWrapper for mobile view (margin) + Avatar Typography + NavBar positioning

## Description

AppWrapper component added.
Avatar style fixed.
Plus Button position fixed.

# Screenshots

Screenshot 2019-11-19 at 20 40 03
Screenshot 2019-11-19 at 20 39 15

### CODE REVIEW

File: src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx [hunk 1]
```diff
@@ -7,11 +7,11 @@ import NEWBIE_SELECT from '../../../graphql/newbieSelect.graphql';
 
 jest.mock('@material-ui/core/Typography', () => 'Typography');
 jest.mock('components/NavBar', () => 'Navbar');
-jest.mock('components/Avatar', () => 'Avatar');
 jest.mock('@material-ui/core/Box', () => 'Box');
 jest.mock('../../PlusButton/', () => 'PlusButton');
 jest.mock('../../Carrousel/', () => 'Carrousel');
 jest.mock('../../BackgroundShape/', () => 'BackgroundShape');
+jest.mock('../../AppWrapper', () => 'AppWrapper');
```
Comment (michaelmoney) [inline src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx]: Could you update all _new_ imports to use absolute paths, introduced by @Tonik1204 ?
I guess you don't have current changes from master in you PR (probably you will get some conflicts) by anyway take a look at master version of this file:
https://github.com/BuildIt-Poland/buddy-app/blob/2e9e864b3e2f3a876d24cd6aa8d3dd98f170450a/src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx#L14

We still have some places where relative path are being used, so it's good to fix those places when we can 🙂

File: src/components/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -94,19 +100,16 @@ const Avatar: React.FC<AvatarProps> = props => {
...
+          <Box fontWeight={theme.typography.fontWeightBold}>{name}</Box>
         </Typography>
       )}
-      {role && (
+      {position && (
         <Typography
+          className={classes[type].position}
           component='h4'
-          variant={type === AvatarType.REGULAR ? 'subtitle1' : 'subtitle2'}>
-          {role}
```
Comment (michaelmoney) [inline src/components/Avatar/Avatar.tsx]: Now it's consistent with the schema field. Good job :clap:

File: src/index.css [hunk 3]
```diff
@@ -1,3 +1,9 @@
+body,
+html,
+#root {
+    height: 100%;
```
Comment (michaelmoney) [inline src/index.css]: Just curious, how this change corresponds to the same style applied in **src/components/AppWrapper/AppWrapper.tsx** ?
```
 root: {
    height: '100%',
    [theme.breakpoints.down('sm')]: {
      paddingTop: theme.spacing(7),
    },
```

Comment (lucianojss) [reply src/index.css]: It's 2 different styles, first in global `.css` is applied only to `body` `html` and `#root` this last one is just the div where react loads the app.
And the `appWrapper root` refers to that specific component. 
I mean to height 100% all parents components also need to have 100% that's why I wrote this. If you have other suggestion to implement that let me know 👍

File: src/components/Avatar/Avatar.tsx [hunk 4]
```diff
@@ -94,19 +100,16 @@ const Avatar: React.FC<AvatarProps> = props => {
...
+        <Typography align='center' component={'h3'} variant='body1'>
+          <Box fontWeight={theme.typography.fontWeightBold}>{name}</Box>
         </Typography>
       )}
-      {role && (
+      {position && (
         <Typography
+          className={classes[type].position}
           component='h4'
-          variant={type === AvatarType.REGULAR ? 'subtitle1' : 'subtitle2'}>
```
[the remark this answers is not in the document]
Comment (lucianojss) [reply src/components/Avatar/Avatar.tsx]: I tried to find in our mock ups but I didn't find any variation for that one, can you tell me where is it? The `subtitle2` is not being used in our typography (as far as I have noticed)

[the remark this answers is not in the document]
Comment (michaelmoney) [reply src/components/Avatar/Avatar.tsx]: I think that `<Avatar type='small'>` is used only in ContactDetails.

>The subtitle2 is not being used in our typography (as far as I have noticed)

I think you removed it here: **[BUD-196] Define/align all typography styles variants**
👉 #24  🤔

File: src/index.css (hunk 3 above)
Comment (michaelmoney) [reply src/index.css]: :+1: OK

File: src/components/Avatar/Avatar.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (lucianojss) [reply src/components/Avatar/Avatar.tsx]: 11997b7a0ba385ae17295aeea58c7a88ded54313

File: src/components/Avatar/Avatar.tsx [hunk 5]
```diff
@@ -106,19 +112,16 @@ const Avatar: React.FC<AvatarProps> = props => {
         </Typography>
       </Grid>
       {name && (
-        <Typography
```
Comment (michaelmoney) [inline src/components/Avatar/Avatar.tsx]: If you wan't to use `h3` variant on `REGULAR` just update it, don't remove the ternary completely.

Comment (lucianojss) [reply src/components/Avatar/Avatar.tsx]: The correct Typography variant for the avatar  is `body1`. The current implementation has `h4` if regular or `h5` if it's small (this small variant doesn't not exist in our design as we already spoke).

So  will update it update the regular avatar (that is the only one we need) to body1 and leave the rest as it was.
`<Typography align='center' component={'h3'}  variant={type === AvatarType.REGULAR ? 'body1' : 'h5'}>`

Comment (michaelmoney) [reply src/components/Avatar/Avatar.tsx]: Yes, thanks :+1: Even though, pls next time raise such things on standup or on our Slack channel, before pushing changes that breaks currently working implementation that you previously accepted during code-review. We wouldn't waste some much discussing on that.  From my perspective I did put some effort to create those _extra_ styles for `<Avatar type='small'` />` thinking that we can re-use it in the future.

Comment (michaelmoney) [reply src/components/Avatar/Avatar.tsx]: **Simplify Avatar component**
https://digitalrig.atlassian.net/browse/BUD-208

I've created separate task ☝️

File: src/components/Avatar/Avatar.tsx [hunk 6]
```diff
@@ -108,17 +114,17 @@ const Avatar: React.FC<AvatarProps> = props => {
       {name && (
         <Typography
           align='center'
-          variant={type === AvatarType.REGULAR ? 'h4' : 'h5'}>
-          <Box fontWeight={theme.typography.fontWeightBold} lineHeight='1.2'>
-            {name}
-          </Box>
+          component={'h4'}
+          variant={type === AvatarType.REGULAR ? 'body1' : 'h5'}>
```
Comment (Tonik1204) [inline src/components/Avatar/Avatar.tsx]: Can we make it as a separate const:
```suggestion
          const type = type === AvatarType.REGULAR ? 'body1' : 'h5';
          ...
          <Typography variant={type}>
```

File: src/components/Carrousel/Carrousel.tsx [hunk 7]
```diff
@@ -17,11 +16,11 @@ const useStyles = makeStyles(theme => ({
     height: '30rem',
     width: '20rem',
     padding: theme.spacing(2),
-    borderRadius: '2rem',
+    borderRadius: '10%',
```
Comment (Tonik1204) [inline src/components/Carrousel/Carrousel.tsx]: 👍

File: src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx [hunk 8]
```diff
@@ -8,11 +8,11 @@ import NewbieSelect from '../NewbieSelect';
 jest.mock('buddy-app-schema', () => {});
```
Comment (Tonik1204) [inline src/components/NewbieSelect/__tests__/NewbieSelect.test.tsx]: It's gonna be unnecessary after the merge of the [PR-39](https://github.com/BuildIt-Poland/buddy-app/pull/39)

File: src/components/Avatar/Avatar.tsx (hunk 6 above)
Comment (michaelmoney) [reply src/components/Avatar/Avatar.tsx]: > Can we make it as a separate const:

Let's leave it as it is. I'm gonna address this here **[Simplify Avatar component](https://digitalrig.atlassian.net/browse/BUD-208)**

**Pull request merged**